index.bs

```diff
@@ -551,25 +597,32 @@ spec:css-syntax-3;
 
   <div algorithm="relevant credential interfaces">
     The <dfn for="CredentialRequestOptions">relevant credential interface objects</dfn> for a given
-    {{CredentialRequestOptions}} (|options|) is a set of [=interface objects=], collected as follows:
+    {{CredentialRequestOptions}} or {{CredentialRequestOptions}} (|options|) is a set of [=interface objects=], collected as follows:
 
     <ol class="algorithm">
-      1.  Let |settings| be the <a>current settings object</a>
+        1.  Let |settings| be the <a>current settings object</a>
+
+        1.  Let |interface objects| be the set of [=interface objects=] on |settings|'
+            [=environment settings object/global object=].
+
+        1.  Let |relevant interface objects| be an [=set/empty=] [=set=].
+
+        1.  For each |object| in |interface objects|:
+
+            1.  If |object| is not a {{Credential}} interface object, and |object|'s [=inherited interfaces=]
```

Comment:
I'm not sure what the reason for this change is. I think the only thing it's doing is adding the `Credential` interface object to the iteration, which will be filtered out in the next step anyway.

Do we even need this step now that we have the registry?

Reply:
It seemed to me that that step had overlooked that |object| itself might be a Credential object, i.e., not just its inherited interfaces. But, yeah, I am probably misunderstanding how this works and so I'll reverse that change.

Yes, I think we need this step in order to filter out objects that do not inherit from Credential; the registry does not figure into that.

done in eed4835